whatshap/cli/polyphase.py

```diff
@@ -526,6 +533,12 @@ def add_arguments(parser):
         action="store_true",
         help=argparse.SUPPRESS,
     )  # help="Verify input genotypes by re-typing them using the given reads.",
+    arg(
+        "--no-mav",
+        default=False,
```

Comment:
You could use `dest="mav", default=True, action="store_false"` to avoid the double negative.

Reply:
Agree. I always used the "store true" approach, probably out of habit. I also made the --no-mav flag public, because there is no reason to hide this from the users.